### ISSUE TITLE
RemovedInSphinx30Warning building docs with sphinx 2.0

### ISSUE THREAD

**Issue by jamadden** [gevent/gevent#1404]
```
doc/mytheme/search.html:20: RemovedInSphinx30Warning: To modify script_files in the theme is deprecated. Please insert a <script> tag directly in your theme instead.
  <input type="text" name="q" value="" />
```

**Comment by jamadden** [comment]
That comes from the search page, which is completely broken ☹️ It refers to javascript files that no longer exist, so, even with javascript properly enabled, it looks like this:
<img width="629" alt="image" src="https://user-images.githubusercontent.com/1256082/56544021-ca698e00-6538-11e9-93ff-668431c6443d.png">

**Comment by jamadden** [comment]
Fixed in #1405
